packages/codegen-test/src/print-test.ts

```diff
@@ -65,25 +65,66 @@ function getConnectionNode(operation: SdkOperation): SdkListField | undefined {
   return operation.model?.fields.list.find(field => field.name === Sdk.NODE_NAME);
 }
 
+/**
+ * Mock value mappings for primitive types
+ */
+const PRIMITIVE_MOCK_VALUES: Record<string, string> = {
+  ["string"]: `"mock-{name}"`,
+  ["string[]"]: `["mock-{name}"]`,
+  ["number"]: `123`,
+  ["number[]"]: `[123]`,
+  ["boolean"]: `true`,
+  ["boolean[]"]: `[true]`,
+};
+
+/**
+ * Get a mock value for an enum type
+ */
+function getMockEnumValue(context: SdkPluginContext, typeName: string, isArray: boolean): string | undefined {
+  const typeWithoutNamespace = typeName.replace(`${Sdk.NAMESPACE}.`, "");
+  const enumDef = context.enums.find(e => e.name.value === typeWithoutNamespace);
+
+  if (enumDef && enumDef.values && enumDef.values.length > 0) {
+    const firstValue = enumDef.values[0].name.value;
+    // GraphQL enum values are uppercase, TypeScript enums use PascalCase
+    const pascalCaseValue = upperFirst(lowerCase(firstValue).replace(/ /g, ""));
```

Comment:
I don't think this is right. Shouldn't it be `startCase(firstValue).replace(/ /g, "")`? As is, it seems like we end up with a lowerCase("helloWorld") -> "hello world" -> "helloworld" -> "Helloworld".